### PR TITLE
Update regexp-prototype-test.html

### PR DESCRIPTION
**Pull request by ackvf**
Seeing two different return values for identical statements seemed hell confusing for an unsuspecting viewer, especially when it is not mentioned anywhere nearby. I think that exposing what is going under the hood is a good introduction and will suggest the viewer to continue reading.

### CODE REVIEW

**Comment by ackvf** [comment]
Excellent!!

May I propose a further simplification? I find it highly distracting and hard to follow, when there are actual code statements mixed with comments. Note that the bare output is still consistent with other examples, like [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
```
var str1 = 'table football';

var regex1 = RegExp('foo*');
var globalRegex1 = RegExp('foo*','g');

console.log(regex1.test(str1));
// true

console.log(regex1.test(str1));
// true

console.log(globalRegex1.lastIndex);
// 0

console.log(globalRegex1.test(str1));
// true

console.log(globalRegex1.lastIndex);
// 9

console.log(globalRegex1.test(str1));
// false
```

**Comment by wbamberg** [comment]
The `// expected output: foo` thing is a convention across all the JS interactive examples, and I'm a fan of consistency. I'm open to this suggestion, but the way to make it happen would be:

1. propose the change in https://discourse.mozilla.org/c/mdn
2. if there's consensus, change the [documented guidelines](https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#indicating-consolelog-output) and update all the examples.

I think this might be an improvement but would need to think more about whether it would work well across all our interactive examples or if there are places where it would be confusing. The nice thing about the current method is that it's very explicit (at the cost of being more wordy).

Also it might take a while to get done, even if everyone agrees, since people are busy :).
